### PR TITLE
Addressed questions about latestSupportedTlsVersion property.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

File: api-reference/beta/resources/subscription.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ A subscription allows a client app to receive notifications about changes to dat
...
+| latestSupportedTlsVersion | String | Specifies the latest version of Transport Layer Security (TLS) that the notification endpoint, specified by **notificationUrl**, supports. The possible values are: `v1_0`, `v1_1`, `v1_2`, `v1_3`. </br></br>For subscribers whose notification endpoint supports a version lower than the currently recommended version (TLC 1.2), specifying this property by a set [timeline](https://developer.microsoft.com/graph/blogs/microsoft-graph-subscriptions-deprecating-tls-1-0-and-1-1/) allows them to temporarily use their deprecated version of TLS before completing their upgrade to TLS 1.2 or higher. For these subscribers, not setting this property per the timeline would result in subscription operations failing. </br></br>For subscribers whose notification endpoint already supports TLS 1.2 or higher, setting this property is optional. In such cases, Microsoft Graph defaults the property to `v1_2`. |
```
Comment (baywet) [inline api-reference/beta/resources/subscription.md]: replace TLC 1.2 by TLS 1.2 (typo)

File: api-reference/v1.0/resources/subscription.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ A subscription allows a client app to receive notifications about changes to dat
...
+| latestSupportedTlsVersion | String | Specifies the latest version of Transport Layer Security (TLS) that the notification endpoint, specified by **notificationUrl**, supports. The possible values are: `v1_0`, `v1_1`, `v1_2`, `v1_3`. </br></br>For subscribers whose notification endpoint supports a version lower than the currently recommended version (TLC 1.2), specifying this property by a set [timeline](https://developer.microsoft.com/graph/blogs/microsoft-graph-subscriptions-deprecating-tls-1-0-and-1-1/) allows them to temporarily use their deprecated version of TLS before completing their upgrade to TLS 1.2 or higher. For these subscribers, not setting this property per the timeline would result in subscription operations failing. </br></br>For subscribers whose notification endpoint already supports TLS 1.2 or higher, setting this property is optional. In such cases, Microsoft Graph defaults the property to `v1_2`. |
```
Comment (baywet) [inline api-reference/v1.0/resources/subscription.md]: same TLC typo

File: api-reference/beta/resources/subscription.md (hunk 1 above)
Comment (angelgolfer-ms) [reply api-reference/beta/resources/subscription.md]: Ok.

**Comment by baywet** [comment]
#signoff

**Pull request merged**